tests/test_kubernetes_master.py

```diff
@@ -98,3 +98,26 @@ def test_service_cidr_expansion():
     unitdata.kv().get.return_value = '10.152.0.0/16'
     update_for_service_cidr_expansion()
     assert kubectl.call_count == 4
+
+
+@mock.patch("reactive.kubernetes_master.get_flags")
```

Comment:
Hrm.  This shows that `get_flags` isn't covered in [charms.unit_test.patch_reactive](https://github.com/juju-solutions/charms.unit_test/blob/master/charms/unit_test.py#L241-L275). Note to myself to fix that gap and maybe refactor this in a future PR (the difference in the test will be pretty trivial anyway).